jobs/build_ova/ansible/main.yml

```diff
@@ -6,6 +6,16 @@
         ip route add default via {{ ova_gateway }} dev {{ ova_net_interface }}
       tags:
         - config-gateway
+      ignore_errors: yes
+      register: gateway
+
+    - name: Enable external network through gateway
```

Comment:
I think it's recommended to have different name for  different step of ansible  ?
the "name" is the same as previous step.